LICENSES.txt

```diff
@@ -58,7 +58,7 @@ Apache-2.0
   gradle-tooling-api-7.3.jar
   gson-2.9.0.jar
   guava-27.0-jre.jar
-  guava-31.0.1-jre.jar
+  guava-32.0.1-jre.jar
```

Comment:
Why do we have double lines for all these dependencies? I s it because we have a transitive guava-27.0-jre.jar (from hadoop-common) that is overridden by guava-32?

Reply:
I believe it is because the LICENSES is for both core and common, so core has 32 and common has 27 (due to Hadoop)

Reply:
Ok, makes sense